### DISCUSSION TITLE
How to access the body?

### DISCUSSION THREAD

**Discussion by fpstidston** [decaporg/decap-cms#6791] [Q&A]
In my import I am getting this, note the missing body in react
```
{
    "attributes": {
        "title": "About",
        "thumbnail": "img/profile.jpg",
        "cats": [
            {
                "name": "Bob",
                "description": "Dig"
            }
        ]
    }
}
```

Whereas in my content file I have this
```
---

title: About
thumbnail: img/profile.jpg
cats:
  - name: Bob
    description: Dig
---
asldaskjda sldjasldakjs dlaksjd laksdj lasd

```

**Comment by Shayan-retrocket** [comment]
It seems that you are dealing with data in different formats. The import you provided is in JSON format, while the content file appears to be in YAML format commonly used in static site generators like Next.js.

To convert the JSON data to JavaScript, you can parse the JSON string using `JSON.parse()`:

```javascript
const data = JSON.parse(jsonString);
```

In this case, `jsonString` should be replaced with the actual JSON string you provided:

```javascript
const jsonString = `{
    "attributes": {
        "title": "About",
        "thumbnail": "img/profile.jpg",
        "cats": [
            {
                "name": "Bob",
                "description": "Dig"
            }
        ]
    }
}`;

const data = JSON.parse(jsonString);
```

After parsing, you can access the data using JavaScript object syntax:

```javascript
const title = data.attributes.title; // "About"
const thumbnail = data.attributes.thumbnail; // "img/profile.jpg"
const cats = data.attributes.cats; // [{ name: "Bob", description: "Dig" }]
```

Remember to adjust the code according to your specific requirements and how you plan to use the data in your React components.

**Comment by fpstidston** [comment]
Please note my questions is about how to access the body text. Which in my examples is "asldaskjda sldjasldakjs dlaksjd laksdj lasd"
